src/app/api/template/[templateId]/complete/route.ts

```
@@ -101,7 +101,7 @@ export async function POST(
 
       // 2. Create the WorkoutSession record with required performanceData
       const templateData = template.workoutData;
-      const totalSets = getTotalSetsCount(template as WorkoutTemplate);
+      const totalSets = getTotalSetsCount(template as any);
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)